library/ssl_tls.c

```diff
@@ -1118,6 +1128,7 @@ static int ssl_set_handshake_prfs( mbedtls_ssl_handshake_params *handshake,
     (void) minor_ver;
     (void) handshake;
 #endif
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
Would it be possible to rather guard the whole function with `#if defined(MBEDTLS_SSL_PROTO_TLS1_2)`:
```C
#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
static int ssl_set_handshake_prfs( ... )
{
...
}
#endif
```
?

Reply:
Not only ssl_set_handshake_prfs. 
Guard mbedtls_ssl_derive_keys